velox/type/Timestamp.cpp

```diff
@@ -166,9 +165,24 @@ void appendSmallInt(int n, std::string& out) {
   out.append(intToStr[n], 2);
 }
 
+std::string::size_type getCapacity(const TimestampToStringOptions& options) {
+  int width = static_cast<int>(options.precision);
+  switch (options.mode) {
+    case TimestampToStringOptions::Mode::kDateOnly:
+      return 10;
+    case TimestampToStringOptions::Mode::kTimeOnly:
+      // hh:mm:ss.precision
+      return 9 + width;
```

Comment:
Nit : Inline static_cast<int>(options.precision) here to avoid use of the width variable. Width variable is misleading for kDateOnly option.

Reply:
I changed the width variable to precisionWidth